mifospay/src/main/java/org/mifos/mobilewallet/mifospay/receipt/ui/ReceiptActivity.java

```diff
@@ -199,9 +199,15 @@ void copyReceiptLink() {
 
     @OnClick(R.id.transaction_reciept_share)
     void shareReceiptLink() {
+        String senderMsg =
+                "Here's the receipt link for the transaction made using MifosPay from payer "
```

Comment:
Avoid hardcoding as usual : )

Reply:
okay, I'll make a string constant for that thanks for reminding i totally forgot:).